docs/apis/streaming/event_timestamps_watermarks.md

```diff
@@ -308,6 +269,89 @@ class TimeLagWatermarkGenerator extends AssignerWithPeriodicWatermarks[MyEvent]
 </div>
 
 
+#### **With Ascending timestamps**
+
+The simplest special case for periodic watermark generation is the case where timestamps within one source occur in ascending order.
+In that case, the current timestamp can always act as a watermark, because no lower timestamps will occur any more.
+
+Note that it is only necessary that timestamps are ascending *per parallel data source instance*. For example, if
```

Comment:
No, it is true. This part of the documentation is not new, it is actually in the 1.0 documentation. 
I just moved it to another section that I think it fits better.